content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -49,11 +52,11 @@ for each product, including which package each item is shipped out in. To use pa
 --> Operations`.
 
 On a delivery order, separate the products into different packages by clicking on the
-:guilabel:`Detailed Operations` icon next to each product. A pop up will appear, allowing the
-amounts to be specified for each product, and what pack(s) the product(s) will go into. Add a line
-for each additional package to be used, and specify the amount of the product to go in the specific
-package. Once all the products for each line has been entered, click :guilabel:`Confirm`. Repeat
-this for all products on the delivery order.
+:guilabel:`Detailed Operations` icon on the right of each product. A pop up will appear, allowing
```

Comment:
Just wanted to add some extra  specificity on where the icon is. You marked it in the image below, but I wanted to make it super clear.

Reply:
Excellent- I agree